From 3f7be623b07894cd6ed443eb19b2ba94beee3efb Mon Sep 17 00:00:00 2001
From: dimitr111adis <t8220036@aueb.gr>
Date: Mon, 25 Nov 2024 17:55:04 +0200
Subject: [PATCH 5/7] Update Form_Groupify.java

---
 Form_Groupify.java | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Form_Groupify.java b/Form_Groupify.java
index 006b7a0..5adbb78 100644
--- a/Form_Groupify.java
+++ b/Form_Groupify.java
@@ -33,12 +33,12 @@ public class Form_Groupify {
 
         System.out.println("8. Προσαρμόζομαι εύκολα σε αλλαγές στα σχέδια που μπορεί να προκύψουν κατά τη διάρκεια της εργασίας");
 
-        System.out.println("9. Πόσα από τα παρακάτω χαρακτηριστικά προσωπικότητας θεωρείτε ότι διαθέτετε: ομαδικότητα, συνέπεια, ηγετική ικανότητα, οργάνωση, διάθεση για ενασχόληση" , 
-         "1 = 1, 2 = 2, 3 = 3, 4 = 4, 5 = 5");
+        System.out.println("9. Πόσα από τα παρακάτω χαρακτηριστικά προσωπικότητας θεωρείτε ότι διαθέτετε: ομαδικότητα, συνέπεια, ηγετική ικανότητα, οργάνωση, διάθεση για ενασχόληση");
+        System.out.println("1 = 1, 2 = 2, 3 = 3, 4 = 4, 5 = 5");
 
-        System.out.println("10. Γενικά είστε ικανοποιημένος/η με την απόδοσή σας ;",
+        System.out.println("10. Γενικά είστε ικανοποιημένος/η με την απόδοσή σας ;" +
          "1 = Καθόλου Ικανοποιημένος/η, 2 = Σχεδόν Ποτέ, 3 = Ουδέτερος/η, 4 = Σχετικά ικανοποιημένος/η, 5 = Απολύτως ικανοποιημένος/η");
-         
+
     }
 }
 
-- 
2.42.0.windows.2

